.github/dependabot.yml

```diff
@@ -11,3 +11,6 @@ updates:
       interval: "daily"
     ignore:
       - dependency-name: "org.apache.maven.plugins:maven-compiler-plugin"
+        # For Quarkus, ignore all minor updates to stay on LTS stream
```

Comment:
Theoretically it could break, practically I don't think it will, but you never know. :-)

``Quarkus`` develops at a very fast pace. I have the fear that it will be some effort to keep this extension in sync. By just supporting the ``LTS`` officially, it makes things easier. Although there is a high chance that this extension will even work with the latest version of ``Quarkus``. In case of a breaking change we can react anyway.

For the moment I would go with the LTS support only. In case the project has some more contributors we could even support the latest and greatest versions officially :-)

Reply:
@wjglerum does it sound fair enough?

Reply:
makes sense, let's start with the LTS first! Im happy to help out btw :)